Brokerages/Bitfinex/BitfinexSymbolMapper.cs

```diff
@@ -24,175 +25,16 @@ namespace QuantConnect.Brokerages.Bitfinex
-            "MNAETH",
-            "FUNBTC",
-            "SPKBTC","SPKETH",
-            "RCNBTC","RCNETH",
-            "RLCETH",
-            "AIDBTC","AIDETH",
-            "SNGBTC","SNGETH",
-            "ELFBTC","ELFETH",
-            "AIOETH",
-            "REQBTC","REQETH",
-            "RDNBTC","RDNETH",
-            "LRCETH",
-            "WAXETH",
-            "AGIBTC","AGIETH",
-            "BFTETH",
-            "MTNBTC","MTNETH",
-            "DTHBTC","DTHETH",
-            "MITBTC","MITETH",
-            "STJBTC","STJETH",
-            "XLMJPY",
-            "XVGEUR","XVGJPY","XVGGBP","XVGETH",
-            "BCIUSD","BCIBTC",
-            "KNCETH",
-            "POABTC","POAETH",
-            "LYMBTC","LYMETH",
-            "UTKBTC","UTKETH",
-            "VEEBTC","VEEETH",
-            "DADUSD","DADBTC","DADETH",
-            "ORSBTC","ORSETH",
-            "AUCBTC","AUCETH",
-            "POYBTC","POYETH",
-            "FSNBTC","FSNETH",
-            "CBTBTC","CBTETH",
-            "ZCNBTC","ZCNETH",
-            "SENUSD","SENBTC","SENETH",
-            "NCABTC","NCAETH",
-            "CNDBTC",
-            "CTXBTC","CTXETH",
-            "PAIBTC",
-            "SEEBTC","SEEETH",
-            "ESSBTC","ESSETH",
-            "ATMBTC","ATMETH",
-            "HOTBTC","HOTETH",
-            "DTABTC","DTAETH",
-            "IQXBTC",
-            "WPRBTC","WPRETH",
-            "ZILBTC","ZILETH",
-            "BNTBTC","BNTETH",
-            "ABSETH",
-            "XRAETH",
-            "MANETH",
-            "BBNUSD","BBNETH",
-            "NIOUSD","NIOETH",
-            "VETETH",
-            "UTNETH",
-            "TKNETH",
-            "CNNETH",
-            "BOXETH",
-            "MGOETH",
-            "RTEETH",
-            "YGGETH",
-            "MLNETH",
-            "WTCETH",
-            "CSXETH",
-            "INTETH",
-            "DRNETH",
-            "WLOXLM",
-            "VLDETH",
-            "ENJETH",
-            "ONLETH",
-            "PASETH",
-            "ZRXDAI",
-            "OMGDAI",
-        };
-
-        /// <summary>
-        /// The list of active Bitfinex symbols.
-        /// </summary>
-        public static List<string> ActiveSymbolStrings =
-            KnownSymbolStrings
-                .Where(x => !DelistedSymbolStrings.Contains(x))
-                .ToList();
-
-        /// <summary>
-        /// The list of known Bitfinex currencies.
-        /// </summary>
-        private static readonly HashSet<string> KnownCurrencies = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
-        {
-            "EUR", "GBP", "JPY", "USD"
-        };
+        public static readonly HashSet<string> KnownSymbolStrings =
+            new HashSet<string>(SymbolPropertiesDatabase
+                .FromDataFolder()
+                .GetSymbolPropertiesList(Market.Bitfinex, SecurityType.Crypto)
+                .Select(x => x.Key.Symbol),
+            StringComparer.OrdinalIgnoreCase);
```

Comment:
Don't think having duplicates could happen, the `SymbolPropertiesDatabase` would throw on static constructor?